apps/prairielearn/src/pages/userSettings/userSettings.html.ts

```diff
@@ -192,6 +222,64 @@ export function UserSettings({
   `.toString();
 }
 
+function PurchaseTable({ purchases }: { purchases: PurchaseRow[] }) {
+  return html`
+    <div class="table-responsive">
+      <table class="table">
+        <thead>
+          <tr>
+            <th>ID</th>
+            <th>Course</th>
+            <th>Date</th>
+            <th>Amount</th>
+            <th>Status</th>
+          </tr>
+        </thead>
+        <tbody>
+          ${purchases.map((purchase) => {
+            const courseName = purchase.course
+              ? `${purchase.course.short_name}: ${purchase.course.title}`
+              : 'Unknown course';
+
+            const courseInstanceName =
+              purchase.course_instance?.long_name ?? 'Unknown course instance';
+
+            return html`<tr>
+              <td>${purchase.stripe_checkout_session.id}</td>
+              <td>
+                <a
+                  ${purchase.course_instance == null
+                    ? ''
+                    : html`href="/pl/course_instance/${purchase.course_instance?.id}"`}
+                >
+                  ${courseName} (${courseInstanceName})
+                </a>
+              </td>
+              <td>${purchase.stripe_checkout_session.created_at?.toString()}</td>
```

Comment:
I should format this better. Is it time to pull over some of our nice formatting utilities from PrairieTest?

Reply:
Probably! Rolling some of them up into another @prairielearn/formatter lib sounds like a good idea.

Reply:
In the interest of landing this, I'm going to avoid doing that for the time being. I switched to `.toUTCString()` in 5cd835c, which is a slight improvement over a plain `.toString()`. The date now shows as something like the following:

> Thu, 17 Aug 2023 20:28:03 GMT

I think this is acceptable for now.

Reply:
Agreed.